examples/with-vercel-kv/app/page.tsx

```diff
@@ -1,4 +1,4 @@
-import { Ratelimit } from "@upstash/ratelimit";
+import { Ratelimit } from "../../../src";
```

Comment:
Is there any particular reason for this import change?

Reply:
Yes this is still a draft PR and I am using a different path for debugging during development.